File: Python/Pattern_d5.py
"""
Patetrn

4 
4 3
4 3 2
4 3 2 1

n = 4

"""
print("Enter the number of rows")
n=int(input())

print("Here is the pattern")

i=n
while i>=1:
    j=n
    while j>=i:
        print(j,end=" ")
        j-=1
    print("\n")
    i-=1

File: patternsJAVA/patternD5.java
/*
Pattern is:
4 
4 3
4 3 2
4 3 2 1
*/
public class patternD5 {
    public static void main(String[] args) {
        int n=4;
        for (int i = n; i >=1; i--) {
            for (int j = n; j >=i; j--) {
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
}
